doc/jemalloc.xml.in

```diff
@@ -1185,6 +1185,40 @@ mallctl("arena." STRINGIFY(MALLCTL_ARENAS_ALL) ".decay",
         enabled.  The default is <quote></quote>.</para></listitem>
       </varlistentry>
 
+      <varlistentry id="opt.stats_interval">
+        <term>
+          <mallctl>opt.stats_interval</mallctl>
+          (<type>uint64_t</type>)
+          <literal>r-</literal>
+        </term>
+        <listitem><para>Average interval between statistics outputs, as measured
+        in bytes of allocation activity.  The actual interval may be sporadic
+        because decentralized event counters are used to avoid synchronization
+        bottlenecks.  When triggered, the
+        <function>malloc_stats_print()</function> function is called.  <link
+        linkend="opt.stats_interval_opts"><mallctl>opt.stats_interval_opts</mallctl></link>
+        can be combined to specify output options.  The default value is 0,
+        which disables this option.</para></listitem>
```

Comment:
I'd clarify that the output may be triggered on any thread (including, say, a jemalloc background thread).

Reply:
Good point. Will update (however probably not mention jemalloc bg thds, as they never actually allocate or go down the event path (their TSD is limited to a non nominal state).